src/app-host/app-host.js

```diff
@@ -72,22 +75,27 @@ socket.on('exec-failure', function (data) {
     }
 });
 
+socket.on('start-live-reload', function () {
+    livereload.start(socket);
+});
+
+telemetry.init(socket);
```

Comment:
Good idea, I will change it to call init when the server requests to, so telemetry never gets initialized if it is disabled.